plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/DBUtils.java

```diff
@@ -72,13 +72,23 @@ public final class DBUtils {
     @NotNull
     public static String getQuotedIdentifier(@NotNull DBPNamedObject object)
     {
-        return object instanceof DBSObject ? getQuotedIdentifier(((DBSObject) object).getDataSource(), object.getName()) : object.getName();
+        if (object instanceof DBSContextBoundAttribute) {
+            return ((DBSContextBoundAttribute) object).formatMemberReference("");
```

Comment:
Can we use `null` instead of empty string?

Reply:
We use `null` here to add default container (table) name before attribute name (PostgreDataBoundTypeAttribute:L65 `DBUtils.getQuotedIdentifier(context.getParentObject()`), but we could add another method parameter to specify whether we need to add container name. Then it will look like `return ((DBSContextBoundAttribute) object).formatMemberReference(false, null);` . Or `formatMemberReference(true, containerName)` in another place of usage. How do you think: it will be better?